package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>cape_ros</name>
-  <version>1.0.0</version>
+  <version>1.0.3</version>
   <description>ROS wrapper for CAPE plane and cylinder extraction</description>
```

Comment:
Since it is being used for only plane extraction, the description should reflect that.

Reply:
We don't use the cylinder extraction but the code still has cylinder extraction, so I think it's OK to keep it.